pysrc/models/block_range.py

```diff
@@ -0,0 +1,27 @@
+"""
+BlockRange Model is just a data class for left and right bounds
+"""
+from dataclasses import dataclass
+
+from dune_client.types import QueryParameter
+
+
+@dataclass
+class BlockRange:
+    """
+    Basic dataclass for an Ethereum block range with some Dune compatibility methods.
+    # TODO - this data class could probably live in dune-client.
```

Comment:
Similar to the two previously mention issues in my opinion it would be nice to have it here too.